src/ngx_http_lua_shdict.c

```diff
@@ -2712,6 +2759,18 @@ ngx_http_lua_ffi_shdict_incr(ngx_shm_zone_t *zone, u_char *key,
 
     ngx_memcpy(p, (double *) &num, sizeof(double));
 
+    if (exptime > 0) {
+        dd("setting expire time to %d", exptime);
+
+        tp = ngx_timeofday();
+        sd->expires = (uint64_t) tp->sec * 1000 + tp->msec
+            + (uint64_t) (exptime * 1000);
+
+    } else if (exptime == 0) {
```

Comment:
Better make `exptime < 0` turn into `exptime == 0` as well. So better replace this `else if` with `else` directly?